kolibri/plugins/coach/assets/src/views/plan/CoachExamsPage/index.vue

```diff
@@ -7,6 +7,29 @@
         :tabsId="PLAN_TABS_ID"
         :activeTabId="PlanTabs.QUIZZES"
       >
+        <div
+          v-if="channels.length === 0"
+          class="alert banner-spacing"
+          :style="{ backgroundColor: $themePalette.yellow.v_200 }"
+        >
+          <div>
+            <KIcon
+              icon="warning"
+              class="warning-icon"
+              :color="$themePalette.yellow.v_1100"
+            />
+          </div>
+
+          <div class="error-message">
+            <p v-if="channels.length === 0">{{ noResourcesAvailable$() }}</p>
```

Comment:
[improvement, non-blocking] To simplify this, you could put the condition on message container div here, rather than on the `<p>` tag and on the link separately. (But keep the `v-if="deviceContentUrl` on the link)